src/SurahTitle/Surahs/102.js

```diff
@@ -1,5 +1,5 @@
 
-import React, { PropTypes } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
`fill={props.color}` was causing some issues last time I was working with the dark theme on quran.com, I wish we could configure fill using CSS :)

Reply:
we can provide some classname which you can use to override the fill?

Reply:
see:
https://github.com/quran/common-components/blob/ae68dc2c9c5eb2e724be84aef338032dbe8ef959/src/SurahTitle/index.stories.js#L12
https://github.com/quran/common-components/blob/ae68dc2c9c5eb2e724be84aef338032dbe8ef959/src/SurahTitle/index.stories.js#L43